frappe/website/js/web_form.js

```diff
@@ -281,14 +281,10 @@ frappe.ready(function() {
 			callback: function(data) {
 				if(!data.exc) {
 					frappe.doc_name = data.message;
-					if(!frappe.login_required) {
```

Comment:
login_required flag should not be required to show success message. When a user is logged in he stays on same page and just saved message appears. Or is it that success message is only for website users?